python/runLumiPixel0SimDPMRestGas.C

```diff
@@ -0,0 +1,200 @@
+#ifndef __CLING__
```

Comment:
I would prefer to add this to PandaROOT instead, all the other run macros are there too.